docker-compose.yml

```diff
@@ -100,7 +100,7 @@ services:
 
           SLEEP_DURATION=$${MINE_INTERVAL}
           BLOCK_HEIGHT=$$(bitcoin-cli -rpcconnect=bitcoind getblockcount)
-          if [ "$${BLOCK_HEIGHT}" -ge "$${STACKS_30_HEIGHT}" ]; then
+          if [ "${BLOCK_HEIGHT}" -ge $(( ${STACKS_30_HEIGHT} - 1 )) ]; then
```

Comment:
Does this work for you locally? I think `$` has to be doubled (to escape) in this yaml context?

Reply:
You're right. I copy/pasted this from the k8s repo and forgot to make that change. Will push the fix shortly.

Reply:
Fixed in 3c7b5a720c4fc2c42bd56737b22cbbcc5f48fe37